deploy/es2k/infraagent-config.yaml

```diff
@@ -2,3 +2,12 @@ interfaceType: cdq
 interface: ens801f0
 mtls: true
 insecure: false
+# Possible debug levels
+#  "Panic"
+#  "Fatal"
+#  "Error"
+#  "Warn"
+#  "Info"
+#  "Debug" (Default)
```

Comment:
Should Debug be default or maybe Error? One of the major reasons for giving out knobs to change log levels is to so that low-level logs are suppressed in production and can be made debug if needed

Reply:
Currently the default level is set to debug for dev and testing purposes and we are not yet production ready. We can change the default level anytime if required.